gtpl/tools/compile.md

```diff
@@ -0,0 +1 @@
+- Unfortunately, this solution does not compile correctly. Did you run the tests? Maybe you accidentally submitted an incomplete solution? If you're having trouble getting your program to compile, you can always ask me for help!
```

Comment:
We could include the error message here, but we don't do that for test failures, so maybe it's better to encourage the student to find the error for themselves.